init.sh

```diff
@@ -112,7 +131,7 @@ then
     echo "DONE"
 
     echo "===> Starting backup..."
-    exec su -pc "mydumper ${CLI_OPTIONS}" ${USER}
+    sudo -u ${USER} mydumper ${CLI_OPTIONS}
```

Comment:
Hmm, I haven't tested this yet. Changing from `exec` to `sudo` was required to be able to call the hooks afterwards, because `exec` will not continue with the execution of this script.